tools/rebuildEM/main.cpp

```diff
@@ -41,6 +40,20 @@ static void usage(const string& pname)
   std::cout << "   -s show extent map and quit" << std::endl;
 }
 
+static bool isYes()
+{
+  std::string confirmation;
+  std::cin >> confirmation;
+  if (confirmation.size() == 0)
+    return false;
+
+  boost::algorithm::to_lower(confirmation);
```

Comment:
JFYI to_lower uses locale internally. To access the locale it touches a mutex that is shared across all to_lower invocations. It is safe to use it here b/c it is a one shoot call but I would discourage from using it.